cli/src/main/java/org/openjdk/skara/cli/HgOpenJDKImport.java

```diff
@@ -36,16 +36,11 @@
 import java.util.logging.*;
 
 public class HgOpenJDKImport {
-    private static void die(Exception e) {
-        System.err.println(e.getMessage());
-        System.exit(1);
-    }
-
     private static Supplier<NoSuchElementException> error(String fmt, Object... args) {
         return () -> new NoSuchElementException(String.format(fmt, args));
     }
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) throws IOException, NoSuchElementException {
```

Comment:
And if you go with a RuntimeException-based one I guess there's no need to declare it here either.

Reply:
Yes, you are totally right, thanks!